### PR TITLE
Feature/inba 956 radio buttons

### PR DESCRIPTION
**Pull request by unprofessional**
#### What does this PR do?
- Replaces the `grommet` based `TaskOptionsRadio` component
- Adds `font-awesome` to the project (else grommet adds it as a dependency, but we plan on removing it, so..)
- runs `yarn eslint` (`yarn stylelint` was clean) **because you should be doing this with every push**, guys

#### Related JIRA tickets:
- https://jira.amida.com/browse/INBA-956

#### How should this be manually tested?
- Log in as a Project Manager
- Assign a user a task
- Go to the Project view and see the Task Matrix
  - Click on the ellipses button:
screen shot 2019-01-07 at 1 02 00 am

- Play with the radio buttons and save and ensure they actually work and everything saves the way it should
  - Ensure the styles make sense (currently they are not centered with the label like it was with grommet [due to different implementation] and there is no hover or currently-selected tactile feedback, but I might fix those as a bonus in this PR soon)
screen shot 2019-01-07 at 1 05 39 am

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +18,12 @@
     },
     "dependencies": {
         "@babel/polyfill": "^7.0.0",
-        "bootstrap": "^3.3.7",
+        "bootstrap": "^4.2.1",
```
Comment (unprofessional) [inline package.json]: Oh yeah. Updated bootstrap here as an artifact from an earlier branch to see if there was any breaking differences.. will need to consult with James and roll back if this should be its own PR for full regression testing etc.

**Comment by morleegan** [comment]
@unprofessional - when you click save after selecting an option, it does not save the one you selected. 

*Note*: it does not do this in the original implementation before you removed indaba. But, we should probably still fix this

**Comment by unprofessional** [comment]
They work for me.

screen shot 2019-01-07 at 11 14 00 am

screen shot 2019-01-07 at 11 15 59 am

I've noticed refreshing issues that do not show the correct information unless you refresh the entire page, but that's out of scope here.

If you're seeing 400s, it's most likely because (like me) you don't have your system message service set up properly and it's simply failing to send notices to the user(s).

Double check and confirm, please.

**Comment by morleegan** [comment]
@unprofessional - the buttons don't stay filled in. after you hit save. the message works

**Comment by unprofessional** [comment]
I'm actually looking into trying to grey them out as we speak but I don't see any meaningful data indicating a task was force completed. 

When inspecting `stages`...
screen shot 2019-01-07 at 11 34 44 am
... there doesn't appear to be anything I can use here.

When inspecting `tasks`...
screen shot 2019-01-07 at 11 35 23 am
... it looks like I *could* use the task status of `completed` to determine this -- however to me `completed` is different than `forcedCompleted`.

Now, I could still grey the options out if James feels like a `completed` task status is sufficient and that we don't need to detect a `forcedCompleted` status. But I don't know the in-depth semantic meaning of Indaba backend logic very well and don't want to make assumptions on my own about it.

If we *want* to use a `forcedComplete` status to determine this, however, it would require a fullstack change.

Waiting for James' input before proceeding.

**Comment by jmfadeley** [comment]
@morleegan @unprofessional At the moment, `completed` is sufficient. It could be beneficial to track an admin's heavy handedness however, especially if this causes issues down the road with task validation (which it might, we risk conflicting business needs if survey questions aren't complete). 

Speak with @katebrittain. Let her decide if we require a stronger business requirement for this.

**Comment by katebrittain** [comment]
@morleegan @jmfadeley completed should be sufficient - we're taking this functionality almost directly from legacy Indaba, and the use case is simply "this stage just isn't going to happen for whatever reason and we need to move on". Might be interesting to mark the task on the Task Matrix with "skipped" or "forced ahead" if it weren't any effort (and I may add to the backlog for exploration later), but I don't think we actually need it

**Comment by unprofessional** [comment]
OK. This is good for another pass, @morleegan @eawest2 @rileyorr et al

- Fixed some styles
  - Centered the button position vertically
  - Implement on hover highlight
- Disabled and greyed out the entire form if task status is `completed`
  - Select dropdown isn't greyed out due to grommet special snowflakeyness... which we're going to replace
- Simply removed the Save button if the task is completed
  - NOTE: I thought about removing the three-dot ellipses button, but figured end-users would complain all like "WHERES MY OPTIONS BUTTON" and would need to be explicitly shown that they no longer have any options

File: src/styles/modals/_task-options-form.scss [hunk 2]
```diff
@@ -1,20 +1,54 @@
 $block-class:'task-options-form';
 
+.#{$block-class} {
+    &__container {
+        &--disabled {
+            color: #d9d9d9 !important;
```
Comment (unprofessional) [inline src/styles/modals/_task-options-form.scss]: A note on the use of `!important` -- it's a controversial topic amongst CSS nerds.

Basically it should be a last resort, but also it needs to be understood what's happening along the cascading uhhh path. If we think of cascading styles as a tree, we can think of a cascading class set as a branching path.

Bear in mind that we're using BEM format for our CSS classes. This means class/state modifiers are at the end of a branch path save for hover/focus/etc cursor states.

In my opinion, it is acceptable to use the `!important` override here. I am welcome to counterpoints, however.
